typedb/common/exception.py

```diff
@@ -74,68 +61,98 @@ def __str__(self):
 class ClientErrorMessage(ErrorMessage):
 
     def __init__(self, code: int, message: str):
-        super(ClientErrorMessage, self).__init__(code_prefix="CLI", code_number=code, message_prefix="Client Error", message_body=message)
+        super(ClientErrorMessage, self).__init__(code_prefix="CLI", code_number=code, message_prefix="Client Error",
+                                                 message_body=message)
 
 
-RPC_METHOD_UNAVAILABLE = ClientErrorMessage(1, "The server does not support this method, please check the client-server compatibility:\n'%s'.")
+RPC_METHOD_UNAVAILABLE = ClientErrorMessage(1, "The server does not support this method, please check "
+                                               "the client-server compatibility:\n'%s'.")
 CLIENT_NOT_OPEN = ClientErrorMessage(2, "The client is not open.")
```

Comment:
Let's delete unused exceptions.

In general, I'd only expect to see error we throw and wrap from the native layer, and errors we throw to prevent issues in the native layer (eg. passing an illegal value).